### PR TITLE
feat: replace nonprofit with charity

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: config/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -68,12 +68,12 @@
...
   "embed-title": "Embedded content",
   "footer": {
-    "tax-exempt-status": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) nonprofit organization (United States Federal Tax Identification Number: 82-0779546)",
+    "tax-exempt-status": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) charity organization (United States Federal Tax Identification Number: 82-0779546)",
     "mission-statement": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world.",
     "donation-initiatives": "Donations to freeCodeCamp go toward our education initiatives, and help pay for servers, services, and staff.",
     "donate-text": "You can {{ <0> }}make a tax-deductible donation here{{ </0> }}.",
     "trending-guides": "Trending Guides",
-    "our-nonprofit": "Our Nonprofit",
+    "our-nonprofit": "Our Chairty",
```
Comment (moT01) [inline config/i18n/locales/english/translations.json]:
```suggestion
    "our-nonprofit": "Our Charity",
```

**Comment by Sboonny** [comment]
is it okay if I edit the test for this PR?

**Comment by scissorsneedfoodtoo** [comment]
@Sboonny, feel free to update the test for this PR. I believe it's just some text that needs to be updated in a file or two.

Once your changes are in, @ mention me and I'll take another look.

**Comment by Sboonny** [comment]
> feel free to update the test for this PR. I believe it's just some text that needs to be updated in a file or two.

Ya, I thought the same, but I couldn't find the strings.

I think the base URL in cypress isn't respected, so it may be falling back to https://www.freecodecamp.org/news/thank-you-for-donating

https://github.com/freeCodeCamp/news/blob/b6661b5c85b6da525ffc2f53d31ee5a9707af890/cypress.config.js#L6

---

I have hard coded the value in the old visit functions to use `http://localhost:8080/news`, and going to see if this works.

https://github.com/freeCodeCamp/news/blob/b6661b5c85b6da525ffc2f53d31ee5a9707af890/cypress/e2e/english/page/structured-data.cy.js#L34

---

@scissorsneedfoodtoo, I have no idea.

**Comment by naomi-lgbt** [comment]
It looks like the issue with the tests is due to fetching [this article](https://www.freecodecamp.org/news/thank-you-for-donating). I've updated the article to say charity, and we can re-run the tests once the next build completes.

**Comment by scissorsneedfoodtoo** [comment]
@naomi-lgbt, thanks for catching that and for updating the live article. You're absolutely right — that the text was coming from Ghost and not somewhere in the code.

Sorry for missing that and sending you on a wild goose-chase, @Sboonny.

I updated the Dockerized English Ghost version of that article to match the live one. I also updated a couple of tests to get things passing again.

In the future we could add a page just to run Cypress tests against. But it's not a high priority, and can be handled in a separate PR.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by Sboonny** [comment]
> In the future we could add a page just to run Cypress tests against. But it's not a high priority, and can be handled in a separate PR

We can check for the existence of `p` elements to make sure that the rendering do happen, and leave the text content to the editorial team, because checking the context create blockage for the dev team, while the context isn't their concern.🤔

**Comment by Sboonny** [comment]
> Sorry for missing that and sending you on a wild goose-chase, @Sboonny.

No worries, it was fun to mess with it

**Comment by scissorsneedfoodtoo** [comment]
Awesome. Thanks for your understanding 👍 

> > In the future we could add a page just to run Cypress tests against. But it's not a high priority, and can be handled in a separate PR
> 
> We can check for the existence of `p` elements to make sure that the rendering do happen, and leave the text content to the editorial team, because checking the context create blockage for the dev team, while the context isn't their concern.🤔

Ideally we'd be able to do just that. And I believe we avoid testing for actual text content whenever possible.

But a big part of the tests we have is to make sure the 11ty builds are giving us metadata similar to what Ghost-served pages are. So descriptions, dates, excerpts, and other meta info needs to be verified.

Unfortunately in this case that caused an issue where the published page's excerpt / meta description showed the word nonprofit when we expected to see the word charity.

Still, we don't have to use the donation page for testing. I just grabbed it from the English API a while back because that's the only page on Ghost we've actually published.

We could publish another page on the English Dockerized Ghost page just for testing those things. It could just have some lorem ipsum text that will never change, and we can run tests against that.

**Pull request merged**